lib/fluent/plugin/out_influxdb.rb

```diff
@@ -55,6 +55,14 @@ def configure(conf)
                                               time_precision: @time_precision,
                                               use_ssl: @use_ssl,
                                               verify_ssl: @verify_ssl
+    
+    existing_databases = @influxdb.list_databases.map { |x| x['name'] }
+    if existing_databases.include? @dbname
+        $log.info 'Dababase ' + @dbname + ' exists'
```

Comment:
I think `info` is ok too. Probably, I can replace message and log `Connection to ... was established successfully`. Because only in this case user will see this message.

`info` is ok, because this message is part of initialization for `fluentd` agent, so, it's ok to print information about how it's going.